CodingGuidelines.md

```diff
@@ -7,6 +7,10 @@ In order to improve team wide readablity and lower onboarding time we agreed to
 The coding guidelines can be found [here](https://csharpcodingguidelines.com/).
 Feel free to download the .PDF for later references.
 
+## IDE
+
+At Triple we are not tied to a single operating system, and so it happens that also among the .NET team there is cross-platform development. In addition to the free choice of IDE, there are quite a few flavors possible. However, in order to reduce startup and onboarding time, we should strive to facilitate cross-platform development as best as possible. This includes the generation of a README file, with instructions for environment specific setups. At the least, it should contain the instructions for the cross-platform editor  [Visual Studio Code](https://code.visualstudio.com/).
+
```

Comment:
Question: Should we make an exception for the `/.vscode/tasks.json` and `/.vscode/build.json` files to improve startup time?

Reply:
Is it startup time or onboard time? Those files are generated right?

Reply:
I think we should make exceptions here when the (build) tasks deviate from the generated ones, to speed up the process of getting projects up and running (so from the online repo to actually running on one's machine).
In my current project I have an example, concerning setting up a host for an Azure function, and attaching the debugger to it.